srcpkgs/kwayland/template

```diff
@@ -1,12 +1,13 @@
 # Template file for 'kwayland'
 pkgname=kwayland
 version=5.63.0
-revision=1
+revision=2
 build_style=cmake
 configure_args="-DBUILD_TESTING=OFF"
 hostmakedepends="kcoreaddons pkg-config extra-cmake-modules qt5-host-tools qt5-tools-devel
  wayland-devel"
 makedepends="qt5-tools-devel MesaLib-devel wayland-devel wayland-protocols"
+depends="xorg-server-xwayland qt5-wayland"
```

Comment:
`xorg-server-xwayland` is not needed for kwayland, if it is dep of anything, than `plasma-workspace` since it explicitly passes `--xwayland` to kwin. Regarding `qt5-wayland` I'm not sure out of my head at where it would be needed as explicit dep, since it is needed everywhere, where we need the wayland qt5 platform plugin, so probably also `plasma-workspace`. Since some applications like spectacle link against kwayland, but can also run fine in non kde x sessions and don't need qt5-wayland there. `Qt5-wayland` is more like a implicit dep we would need for all wayland sessions.

Reply:
On one hand I like the approach arch took, with having the Wayland session it it's own package, but on the other hand we don't have any way to split a package properly afterwards without running into the issue of breaking systems. Or we could, and just set it as a dep in the kde5 meta package.

Reply:
Sorry to add addiction to the wrong place. I am new and thought this package gives a KDE Plasma Wayland session.